jenny/templates/deployment.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "jenny.fullname" . }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels:
+    {{- include "jenny.labels" . | nindent 4 }}
+spec:
+  selector:
+    matchLabels:
+      {{- include "jenny.selectorLabels" . | nindent 6 }}
+  {{- if not .Values.autoscaling.enabled }}
+  replicas: {{ .Values.replicaCount }}
+  {{- end }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "jenny.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- include "django.imagePullSecrets" . | nindent 6 }}
+      securityContext:
+        {{- toYaml .Values.securityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command:
+            {{- toYaml .Values.django.command | nindent 12 }}
+          resources:
+            {{- toYaml .Values.resources | nindent 12 }}
+          livenessProbe:
+            httpGet:
+              path: /admin/login/
+              port: {{ .Values.django.port }}
+            initialDelaySeconds: 15
+            periodSeconds: 30
+          readinessProbe:
+            httpGet:
+              path: /admin/login/
+              port: {{ .Values.django.port }}
+            initialDelaySeconds: 5
+            periodSeconds: 5
+          {{- include "jenny.envs" . | nindent 10 }}
+          volumeMounts:
+            - name: {{ .Values.volumes.static.name }}
+              mountPath: /app/static
+            - name: {{ .Values.volumes.media.name }}
+              mountPath: /app/media
+      volumes:
+        - name: {{ .Values.volumes.static.name }}
+          persistentVolumeClaim:
+            claimName: {{ .Values.volumes.static.claimName }}
+        - name: {{ .Values.volumes.media.name }}
+          persistentVolumeClaim:
+            claimName: {{ .Values.volumes.media.claimName }}
```

Comment:
if `.Values.persistence.enabled ` is not enabled, this I will fail I think. You should not declare `volumes` is persistent is not enabled.

Reply:
resolved by https://github.com/openfun/charts/pull/4/commits/0c088965a52f6c2495f65b3001d1a53e50797f6d